.circleci/config.yml

```diff
@@ -1820,14 +1837,9 @@ workflows:
       - js_coverage
 
   nightly:
-    unless: << pipeline.parameters.run_package_release_workflow_only >>
-    triggers:
-      - schedule:
-          <<: *main_only
-          cron: "0 20 * * *"
+    when: << pipeline.parameters.run_nightly_workflow >>
```

Comment:
so... we are not getting nightlies automatically anymore? It would break the workflow of partners who relies on the nightlies to check if there are API changes.
What if we duplicate the nightlies workflow and we keep the cron trigger for one copy and we add the possibility to manually trigger a nightly?

Reply:
See the other comment